### ISSUE TITLE
unwrap() is causing the main thread to panic, and SerdeError

### ISSUE THREAD

**Issue by K0dikas** [Maxuss/chatgpt_rs#2]
By using your example, instead of using `unwrap()`, I have decided to paste the session key in the `ChatGPT::new("SESSION_KEY_HERE")`, to avoid panicking. 

The program did not produce any errors until running it for about 2-5 seconds. It led me to another error; `Error: SerdeError(Error("invalid type: null, expected struct ConversationResponse", line: 1, column: 4))`

**Comment by Maxuss** [comment]
It seems the authentication method was updated, I'll look into it later when I have time

**Comment by jellydn** [comment]
Just FYI from https://github.com/transitive-bullshit/chatgpt-api#session-tokens
```
Update December 12, 2022
Yesterday, OpenAI added additional Cloudflare protections that make it more difficult to access the unofficial API.
```

**Comment by Maxuss** [comment]
Yep, I've seen that. Already implementing a fix: https://github.com/Maxuss/chatgpt_rs/pull/3

**Comment by Maxuss** [comment]
Fixed in #3
